index.html

```diff
@@ -25,5 +25,125 @@ <h2 class="headerSubtitle">LET'S CONNECT</h2>
       <li class="headerIcon"><img src="./img/Fingers.png" alt="fingers"></li>
     </ul>
   </header>
+  
+  <section class="section1">
+    <div class="sectionCard">
+      <img src="./img/card1.png" alt="card1">
+      <h1 class="sectionTitle">Tonic</h1>
+      <div class="sectionCanopy">
+        <h2 class="text1">CANOPY</h2>
+        <img class="counter" src="./img/Counter.png" alt="counter">
+        <h2 class="text2">Back End Dev</h2>
+        <img class="counter" src="./img/Counter.png" alt="counter">
+        <h2 class="text2">2015</h2>
+      </div>
+      <p class="sectionParra">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="sectionTech">
+        <li class="tech">html</li>
+        <li class="tech">css</li>
+        <li class="tech">javascript</li>
+      </ul>
+      <p class="sectionButton">See Project</p>
+    </div>
+    <div class="sectionCard">
+      <img src="./img/card2.png" alt="card2">
+      <h1 class="sectionTitle">Multi-Post<br>Stories</h1>
+      <div class="sectionCanopy">
+        <h2 class="text1">CANOPY</h2>
+        <img class="counter" src="./img/Counter.png" alt="counter">
+        <h2 class="text2">Back End Dev</h2>
+        <img class="counter" src="./img/Counter.png" alt="counter">
+        <h2 class="text2">2015</h2>
+      </div>
+      <p class="sectionParra">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="sectionTech">
+        <li class="tech">html</li>
+        <li class="tech">css</li>
+        <li class="tech">javascript</li>
+      </ul>
+      <p class="sectionButton">See Project</p>

```

Comment:
Kindly, I suggest you implement a link or a button here, because very soon you will use it to show the detail of your work

Reply:
Good job following the advice of the previous reviewer